user-guide/install/before.rst

```diff
@@ -120,6 +122,32 @@ Below is an example port map for interconnection of two *Danube Cloud* nodes (on
 :download:`Example port map in XLSX format (Excel spreadsheet) <portmap.xlsx>`.
 
 
+.. _bios:
+
+BIOS Configuration
+##################
+
+The following settings should be configured in BIOS configuration of your compute node (if available):
+
+- Enable hardware virtualization (KVM) support.
+
+    .. warning:: Hardware virtualization (KVM) support must enabled on the head node.
+
+- Disable NUMA/Node interleaving. If not disabled the follow message may appear during boot time:
+
+    .. code-block:: text
+
+        WARNING: Couldn't read ACPI SRAT table from BIOS. lgrp support will be limited to one group.
+
+- Disable CPU C-States.
+
+    .. note:: Some Intel® processors, which are using the C-States feature, can cause an error that may seriously endanger the run of a working compute node. The error is treated in the system, but you are advised to disable C-States in the BIOS configuration.
```

Comment:
Some Intel® processors that are using the C-States feature, can cause an error that may seriously endanger the correct run of a compute node. The error is treated in the system but you are advised to disable C-States in the BIOS configuration.